tests/integration/lifecycle/test_lifecycle.py

```diff
@@ -369,7 +369,7 @@ def setup(self, new_dir, partitions):
             parts,
             application_name="test_update",
             cache_dir=new_dir,
-            arch="aarch64",
+            arch="amd64",
```

Comment:
This is a behavioural change on the tests (arm64 -> amd64). I'm not sure it matters much, but I figured I'd call it out for examination.

Reply:
Ooo ops this is a typo. Thanks for spotting it.